src/api_manager/fetch_metadata.cc

```diff
@@ -96,7 +96,7 @@ void GlobalFetchGceMetadata(std::shared_ptr<context::GlobalContext> context,
         if (status.ok()) {
           status = context->gce_metadata()->ParseFromJson(&body);
         } else {
-          status = Status(Code::INTERNAL, kFailedMetadataFetch);
+          status = Status::OK;
         }
 
         // update fetching state
```

Comment:
remove "FAILED" state.  Always set to FETCHED.

Reply:
Done